src/objects/nodes.ts

```diff
@@ -1,46 +1,63 @@
-import { UseSubscription, useSubscription } from '@prefecthq/vue-compositions'
-import { watch } from 'vue'
-import { RunGraphConfig, RunGraphFetch } from '@/models/RunGraph'
+import { Graphics } from 'pixi.js'
+import { RunGraphNode } from '@/models/RunGraph'
 import { waitForConfig } from '@/objects/config'
 import { emitter } from '@/objects/events'
 import { waitForScales } from '@/objects/scales'
 import { waitForViewport } from '@/objects/viewport'
-import { effectScopeFactory } from '@/utilities/effectScope'
+import { graphDataFactory } from '@/utilities/graphDataFactory'
 
-let subscription: UseSubscription<RunGraphFetch> | null = null
+const { fetch: getData, stop: stopData } = graphDataFactory()
+const nodes = new Map<string, Graphics>()
```

Comment:
The way I understand it is since we added graphics to the viewport we either need to A, remove the old graphics and draw new ones (slow) or redraw the existing graphics (faster). So keeping a reference to the graphics for each node means we can redraw them even though they've already been added to the viewport.

Reply:
My rough understanding. As long as the class is still in existence there is a texture and a sprite (graphics() = some utils + texture + sprite) for it whether it's on the screen or not. Actually it may still exist even if the class goes away if it hasn't been cleaned up enough. 

It's like a 2d sidescroller game. You'll have textures loaded, things at the ready for when they enter the scene (a gun blast graphic graphic or something).

Reply:
The specific implementation here is also likely to change. I'm focused more on the patterns and logic than the actual  graphics/sprites themselves at this point.